download/index.md

```diff
@@ -99,7 +99,7 @@ You can find a [listing of all previous Redis releases](https://download.redis.i
 
 ### RedisInsight
 
-RedisInsight is a powerful tool for visualizing and optimizing data in Redis or Redis Stack. Read the latest [RedisInsight release notes](https://docs.redis.com/staging/release-ri-v3.0/ri/release-notes/).
+RedisInsight is a powerful tool for visualizing and optimizing data in Redis or Redis Stack. Read the latest [RedisInsight release notes](https://github.com/RedisInsight/RedisInsight/releases/tag/2.0.6).
```

Comment:
I believe it is better to use the following link so not to update it with each release
https://github.com/RedisInsight/RedisInsight/releases